dags/atd_knack_secondary_signals.py

```diff
@@ -0,0 +1,58 @@
+# test locally with: docker compose run --rm airflow-cli dags test atd_knack_secondary_signals
+
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "KNACK_APP_ID": {
+        "opitem": "Knack AMD Data Tracker",
+        "opfield": f"production.appId",
+    },
+    "KNACK_API_KEY": {
+        "opitem": "Knack AMD Data Tracker",
+        "opfield": f"production.apiKey",
+    },
+}
+
+with DAG(
+    dag_id="atd_knack_secondary_signals",
+    description="Update traffic signal records with secondary signal relationships.",
+    default_args=DEFAULT_ARGS,
+    schedule_interval="25 2 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    dagrun_timeout=duration(minutes=60),
```

Comment:
you'll want to remove this and add `execution_timeout` as a default arg. 60 minutes is prob too long eh?